### PR TITLE
MTM-43476 Review alarm mapping

### PR DESCRIPTION
**Pull request by Airdroplet**
* expaded description in few places

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/administration-bundle/business-rules.md [hunk 1]
```diff
@@ -30,14 +30,16 @@ For each alarm mapping, the alarm severity, the alarm type and a description (op
 4. Select the desired new severity, or select "Drop" to not show the alarm at all.
 5. Click **Save** to save your settings.
 
-> **Info:** The alarm type set in an alarm mapping is interpreted as &quot;&#60;type&#62;*&quot;. If you create for example an alarm mapping to address alarms of type &quot;crit-alarm&quot;, the mapping is effective for any type of alarm that starts with this value e.g. &quot;crit-alarm-1&quot;, &quot;crit-alarm-2&quot;, or &quot;crit-alarm-xyz&quot;.
+> **Info:** The alarm type provided as an alarm mapping is interpreted as alarm type prefix: &quot;&#60;type-prefix&#62;*&quot;. If you create for example an alarm mapping to address alarms of type &quot;crit-alarm&quot;, the mapping is effective for any type of alarm that starts with this value e.g. &quot;crit-alarm-1&quot;, &quot;crit-alarm-2&quot;, or &quot;crit-alarm-xyz&quot;.
```
Comment (MWindel) [inline content/users-guide/administration-bundle/business-rules.md:33]:
```suggestion
> **Info:** The alarm type provided as an alarm mapping is interpreted as alarm type prefix: &quot;&#60;type-prefix&#62;*&quot;. If you create, for example, an alarm mapping to address alarms of type &quot;crit-alarm&quot;, the mapping is effective for any type of alarm that starts with this value e.g. &quot;crit-alarm-1&quot;, &quot;crit-alarm-2&quot;, or &quot;crit-alarm-xyz&quot;.
```

File: content/users-guide/administration-bundle/business-rules.md [hunk 2]
```diff
@@ -30,14 +30,16 @@ For each alarm mapping, the alarm severity, the alarm type and a description (op
...
 
 #### To edit an alarm mapping
 
 Expand an alarm mapping to edit it. You may modify the description and the alarm severity. The alarm type is not editable.
 
+> **Info:** To discard modified alarms without save in order to start over, refresh the list.
```
Comment (MWindel) [inline content/users-guide/administration-bundle/business-rules.md:39]:
```suggestion
> **Info:** Refresh the list to discard the modified alarms in order to start over without saving the current status. 
```

Comment (BeateRixen) [reply content/users-guide/administration-bundle/business-rules.md:39]: The sounds very complicated :-). I would just have it: "Refresh the list to discard any changes without saving."

Comment (BeateRixen) [reply content/users-guide/administration-bundle/business-rules.md]: Changed in the source file.

**Pull request merged**